ciphers/affine_cipher.py

```diff
@@ -3,32 +3,12 @@
 
 from . import cryptomath_module as cryptomath
 
-SYMBOLS = (
+SYMBOLS: str = (
     r""" !"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\]^_`"""
     r"""abcdefghijklmnopqrstuvwxyz{|}~"""
 )
 
 
-def main():
```

Comment:
Why remove this interactive demonstration of how the algorithm work.  These sorts of demonstrations invite experimentation.